### PR TITLE
update Simplified Chinese

### PR DESCRIPTION
**Pull request by Lzw104522773**
update Simplified Chinese

### CODE REVIEW

File: RoT/setup-RoT.tp2 [hunk 1]
```diff
@@ -53,7 +53,7 @@ ALWAYS
 			    "schinese" => "CP936"
 			END
 			ACTION_DEFINE_ARRAY charsetsReloadArray BEGIN setup scripts journal END
-			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup-ee END
+			ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN setup setup-ee END
```
Comment (Roberciiik) [inline RoT/setup-RoT.tp2:56]: Hey, this is incorrect.  The `setup.tra` file has to be converted for EE engine, as it is encoded in local code pages.

**Review by Roberciiik — CHANGES_REQUESTED** [review]

**Comment by Lzw104522773** [comment]
I re-upload [setup-RoT.tp2

File: RoT/Worldmap/schinese/worldmap.tra [hunk 2]
```diff
@@ -0,0 +1,41 @@
+
+@ROT_785001 = "阿拉克斯"
```
Comment (Roberciiik) [inline RoT/Worldmap/schinese/worldmap.tra:2]: @Lzw104522773 
Also I am pretty sure that this file should be encoded in your local language encoding.

**Comment by Lzw104522773** [comment]
I re-upload worldmap.tra,and worldmap.tra use ANSI encode. Thanks.

**Review by Roberciiik — APPROVED** [review]

**Pull request merged**